cheroot/server.py

```diff
@@ -1804,6 +1804,7 @@ def serve(self):
                     traceback=True,
                 )
 
+        self._connections.close()
```

Comment:
moving this call inside the thread that is running the selector to ensure the selector is only ever accessed from a single thread

Reply:
This probably needs a code comment